io.github.steamdeck.yml

```diff
@@ -0,0 +1,53 @@
+app-id: io.github.steamdeckutilities
+runtime: org.gnome.Platform
+runtime-version: '43'
+sdk: org.gnome.Sdk
+command:  launcher.sh
+finish-args:
+  - '--share=ipc'
+  - '--socket=x11'
+  - '--filesystem=host'
+  - '--device=dri'
+  - '--share=network'
+  - '--talk-name=org.freedesktop.Flatpak'
+build-options:
+  env:
+    - GOBIN=/app/bin
+    - GOROOT=/app/lib/sdk/golang/
+modules:
+  - name: golang
+    buildsystem: simple
+    sources:
+      - type: archive
+        only-arches:
+          - aarch64
+        url: https://go.dev/dl/go1.20.1.linux-arm64.tar.gz
+        sha256: fc0aa29c933cec8d76f5435d859aaf42249aa08c74eb2d154689ae44c08d23b3
+      - type: archive
+        only-arches:
+          - x86_64
+        url: https://go.dev/dl/go1.20.1.linux-amd64.tar.gz
+        sha256: 000a5b1fca4f75895f78befeb2eecf10bfff3c428597f3f1e69133b63b911b02
+    build-commands:
+      - install -d /app/lib/sdk/golang
+      - cp -rpv * /app/lib/sdk/golang/
```

Comment:
We need go 1.20, but the SDK extension was only providing go 1.19. It may be  there was something I was missing.